guide/Text/Embed/example.md

```diff
@@ -1,3 +1,46 @@
 # Complete Embed
-Pls check this Embed functions first
-...
\ No newline at end of file
+
+Here is a code example of a complete embed.
+
+## Function Format
+
+$title[Title;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$author[Custom Command;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$image[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$description[Description]
+$thumbnail[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$color[#0c2238]
+$footer[Footer]
+
```

Comment:
there's other missing ones, like $selectMenu, $button

Reply:
hopefully got everything in next update